azure-pipelines.yml

```diff
@@ -34,7 +36,7 @@ extends:
         enabled: true
       codeql:
         ${{ if startsWith(variables['Build.SourceBranch'], 'refs/heads/release/') }}:
-          enabledOnNonDefaultBranches: true
+            enabledOnNonDefaultBranches: true
```

Comment:
Since this is yml - is the whitespace change correct? (I guess it will get validated once we merge it)